etc/env/example.env

```diff
@@ -314,3 +321,16 @@ CANCOGEN_FEDERATION_URL=http://federation-service:4232/federation/search
 CANDIG_DATA_PORTAL_VERSION=v0.1.0
 CANDIG_DATA_PORTAL_PORT=2543
 CANDIG_DATA_PORTAL_URL=http://${CANDIG_DOMAIN}:${CANDIG_DATA_PORT}/data-portal
+
+# graphql-interface
+GRAPHQL_CONDA_SOURCE=condaforge
+GRAPHQL_CONDA_TYPE=miniforge3
+GRAPHQL_CONDA_VERSION=latest
+GRAPHQL_INTERFACE_VERSION=v1.0.0
+GRAPHQL_INTERFACE_PORT=7999
+
+GRAPHQL_KATSU_API=http://candigv2_tyk_1:8080/katsu/api
+GRAPHQL_CANDIG_SERVER=http://candigv2_tyk_1:8080/candig
+GRAPHQL_BEACON_ID=com.candig.graphql
+GRAPHQL_KATSU_TOKEN_KEY=authorization
```

Comment:
Please see request for changes in https://github.com/CanDIG/GraphQL-interface/pull/11 regarding file secrets

Reply:
Fixed: See https://github.com/CanDIG/GraphQL-interface/pull/11#discussion_r819079892